workflows/subclustering/🍒_RelevanceAI_Subclustering.ipynb

```diff
@@ -0,0 +1,17241 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Please move all variables to the top 
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/RelevanceAI/workflows/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>